tests/vmi_multus_test.go

```diff
@@ -884,7 +884,8 @@ func cidrToIP(cidr string) string {
 	return ip.String()
 }
 
-func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAddress, prompt string) {
+func configInterface(vmi *v1.VirtualMachineInstance, interfaceNameOrMac, interfaceAddress, prompt string) {
```

Comment:
This is nice workaround, although its may confuse other users and lead to misuse of this function because it can take both interface name or interface mac address w/o distinction of a type ( we pass string and assume it is name or mac).
What you think about creating another function `configureSRIOVInterface()`?

Reply:
Done